package.json

```
@@ -6,8 +6,9 @@
   },
   "private": true,
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
+    "dev": "npm run test && next dev",
+    "build": "npm run test && next build",
```

Nop ça on modifie pas, ça sera au rôle de la CI d'executer test puis build ect. 

Et pour la commande dev ça rend bloquant le dev c'est chiant. 

D'une manière générale on préfère run les tests via l'IDE et activer le hot reload.

```suggestion
    "dev": "next dev",
    "build": "next build",
```